pkg/cli/tag/tag.go

```diff
@@ -302,7 +305,7 @@ func isCrossImageStream(namespace string, srcRef imagev1.DockerImageReference, d
 }
 
 // Validate validates all the required options for the tag command.
-func (o TagOptions) Validate() error {
+func (o *TagOptions) Validate() error {
```

Comment:
What is the reason of this change with respect to the new flag?. If just for standardization, could you please also change this one https://github.com/openshift/oc/blob/f372c323adfdac963c09af0dda2745401c3b1c96/pkg/cli/tag/tag.go#L376?

Reply:
I changed this, because the Validate method now actively sets the option to "Legacy" in case it is missing (as opposed to leaving the fallback to the API server after the resource is created): https://github.com/openshift/oc/blob/f372c323adfdac963c09af0dda2745401c3b1c96/pkg/cli/tag/tag.go#L367

This is the same as in the respective change in https://github.com/openshift/oc/pull/1289/files

From what I've seen, the way option structs are passed to these method is not really standardized in the oc code base. So my intuition was to only do this if the method actually changes the struct.

Reply:
Good point, you are right